components/camera/camera.go

```diff
@@ -164,9 +167,16 @@ func (ni *NamedImage) MimeType() string {
 	return ni.mimeType
 }
 
-// ImageMetadata contains useful information about returned image bytes such as its mimetype.
+// Annotations returns the annotations of the NamedImage.
+func (ni *NamedImage) Annotations() data.Annotations {
```

Comment:
any scenarios where RDK may want to set annotations after initialization?

Reply:
I'm not sure. You can add this in post-processing as well, so after the image has been uploaded